docs/developers/json-rpc/geth_js_console.md

```diff
@@ -0,0 +1,256 @@
+<!--
+order: 6
+-->
+
+# Geth JavaScript Console Guide
+
+Go-ethereum responds to instructions encoded as JSON objects as defined in the [JSON-RPC-API](https://geth.ethereum.org/docs/rpc/server). To perform and test these instructions, developers can use tools like curl. However, this is a low level and rather error-prone way to interact with the node. Most developers prefer to use convenient libraries that abstract away some of the more tedious and awkward tasks such as converting values from hexadecimal strings into numbers, or converting between denominations of ether (Wei, Gwei, etc). One such library is Web3.js. The purpose of Geth’s Javascript console is to provide a built-in environment to use a subset of the Web3.js libraries to interact with a Geth node. You can use this powerful tool to interact with an Evmos node too! Use this guide to learn how to set up and use the Geth JS console with an Evmos node. {synopsis}
+
+## Pre-requisite Readings
+
+- [Go-ethereum CLI](https://geth.ethereum.org/docs/interface/javascript-console) {prereq}
+- [Evmos (local) node](https://docs.evmos.org/developers/localnet/single_node.html) {prereq}
+
+### Installing Go-ethereum JavaScript Console
+
+Install the Go-ethereum JavaScript Console (`geth`) following the procedure corresponding to your OS in the [geth docs](https://geth.ethereum.org/docs/install-and-build/installing-geth).
+
+Check that the installation was successful by running the following command:
+
+```
```

Comment:
you can use the [Markdown code block format](https://markdown.land/markdown-code-block) to highlight the code. I suggest applying this to all the other blocks too

```suggestion
```bash
```